processor.go

```diff
@@ -523,10 +523,27 @@ func (g *Processor) newStorage(topic string, id int32, update UpdateCallback) (*
 		}, nil
 	}
 
-	st, err := g.opts.builders.storage(topic, id)
+	wg := new(sync.WaitGroup)
+	var err error
+	var st storage.Storage
+	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
let's have a smaller ticker, like every 10 seconds? We wanted to avoid having to wait too long until we see anything

Reply:
But because this takes hours it would pollute the log thats why I used a rather big ticker.